docs/contributing/platform_api/app/viewsets.rst

```diff
@@ -3,6 +3,8 @@ pulp.app.viewsets
 
 All viewsets documented here should be imported directly from the ``pulp.app.viewsets`` namespace.
 
+Viewsets that dispatch tasks have to return a :class:`pulp.app.response.OperationPostponedResponse`.
```

Comment:
I believe you can use a `~` to cause autodoc to shorten the name to just `OperationPostponedResponse`. For example:  https://github.com/celery/kombu/blob/3.0/kombu/transport/qpid.py#L54